custom/icds_reports/utils/aggregation.py

```diff
@@ -1322,3 +1323,106 @@ def indexes(self, aggregation_level):
             indexes.append('CREATE INDEX ON "{}" (supervisor_id)'.format(self.tablename))
 
         return indexes
+
+
+class AwcInfrastructureAggregationHelper(BaseICDSAggregationHelper):
+    ucr_data_source_id = 'static-infrastructure_form_v2'
+    aggregate_parent_table = AGG_INFRASTRUCTURE_TABLE
+    aggregate_child_table_prefix = 'icds_db_infra_form_'
+    column_names = (
+        'timeend',
+        'awc_building', 'source_drinking_water', 'toilet_functional',
+        'electricity_awc', 'adequate_space_pse',
+        'adult_scale_available', 'baby_scale_available', 'flat_scale_available',
+        'adult_scale_usable', 'baby_scale_usable', 'cooking_utensils_usable',
+        'infantometer_usable', 'medicine_kits_usable', 'stadiometer_usable',
+    )
+
+    def _window_helper(self, column_name):
```

Comment:
since this type of expression is used in a few other of these aggregation helpers, this might be nice to pull out and apply there, but not blocking